src/pat/structure/js/views/app.js

```diff
@@ -78,7 +78,13 @@ export default BaseView.extend({
             url: this.options.collectionUrl,
         });
 
-        this.setAllCookieSettings();
+        // initialize batch size
+        this.collection.paginator_ui.perPage = parseInt(this.getCookieSetting("perPage", 15))
+
+        this.activeColumns = this.getCookieSetting(
+            this.activeColumnsCookie,
+            this.activeColumns
+        );
```

Comment:
By removing `this.collection.howManyPer` in the `this.setAllCookieSettings` we do not have duplicated calls to the `render` method anymore. 

/cc @agitator might wanna test this?